getting_started/publish_app.md

```diff
@@ -0,0 +1,108 @@
+
+![](../images/approval-app-rejected.png)
+
+If your application is approved, you are ready to move on to the [next steps](#next-steps) found at the end of this document.
+
+### Applications for publication
+
+In order for an application to be available for public users, it must first be approved by the Adobe Review team. To begin, navigate to the *Project overview* for the project containing the application that you wish to publish.
+
+![](../images/approval-empty-project.png)
+
+Select **Approval** from the left navigation to view the **App Submission Details**. These details include information about the application that will be shown to end users and the Adobe Review team.
+
+Once the submission details have been completed, select **Submit** to begin the approval process.
+
+![](../images/approval-personal-app-submission-details.png)
+
+At that time your application *Status* will be updated to "In Review" and the **App Submission Details** that you filled in previously will be visible, however they will be greyed out and you will be unable to edit them. 
+
+![](../images/approval-personal-app-in-review.png)
+
+### Published app
+
+After your application has been submitted for approval, the admin could see your app pending for review in Adobe Exchange. next section will describe admin approval flow from My exchange. From console side, once an application has been approved, either by internal reviewers or the Adobe Review team, its *Status* will be updated to "Published" and the application will be available for use either by employees within your enterprise organization (for Project Firefly applications) or for the general public through Adobe Exchange.
+
+![](../images/approval-published.png)
+
+## 2: Administrater review your app (MyExchange Part)
+
+Once an application has been submitted for approval, either internal reviewer or Adobe Review team will see an app in My Exchange -> Experience Cloud Apps, you will see your submitted apps under the pending review session。
+
+![](../images/approval-myexchange.png)
+
+The reviewer could review this app, either approve it or reject it 
+
+![](../images/approval-myexchange-review.png)
+
+After approve the app, you will see the app in the approved section, the reviewer could revoke this app to unpublished the app. The owner of the app could re-submit the application for review.
+
+![](../images/approval-myexchange-revoke.png)
+
+
+## 3: Check your published app at Experience Cloud
+
+Once the reviewer approve your application, you will received an email notification. and your app will show up at Experiences cloud
+
+![](../images/approval-myapp-home.png)
+
+Click into `Project Firefly App` you will see your published app
```

Comment:
Click on `Project Firefly Apps` to discover the applications published for your organization.